site/docs/components/static-list/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Static list
+data:
+  description: "The Static list component is a versatile and essential UI element used to display a fixed set of items in a structured and organised manner. This component is ideal for presenting non-interactive lists where the content does not require user input or dynamic updates. It is one of the building block to create the file upload pattern. We advise you to use this as the list item for the same pattern."
```

Comment:
This sentence has too many parts to it and could be more concise.

"The StaticList component manages the layout of non-interactive list items, providing a structured and composable way to display content. It is ideal for use in patterns like file upload or checkout baskets."